lms/envs/devstack.py

```diff
@@ -386,20 +386,20 @@ def should_show_debug_toolbar(request):
     from .private import *  # pylint: disable=import-error,wildcard-import
 
 ########################## THEMING  #######################
-# If you want to enable theming in devstack, uncomment this section and add any relevant
-# theme directories to COMPREHENSIVE_THEME_DIRS
-
 # We have to import the private method here because production.py calls
 # derive_settings('lms.envs.production') which runs _make_mako_template_dirs with
 # the settings from production, which doesn't include these theming settings. Thus,
 # the templating engine is unable to find the themed templates because they don't exist
 # in it's path. Re-calling derive_settings doesn't work because the settings was already
 # changed from a function to a list, and it can't be derived again.
 
-# from .common import _make_mako_template_dirs
-# ENABLE_COMPREHENSIVE_THEMING = True
-# COMPREHENSIVE_THEME_DIRS = [
-#     "/edx/app/edxapp/edx-platform/themes/"
-# ]
-# TEMPLATES[1]["DIRS"] = _make_mako_template_dirs
-# derive_settings(__name__)
+from .common import _make_mako_template_dirs
+ENABLE_COMPREHENSIVE_THEMING = True
+COMPREHENSIVE_THEME_DIRS = [
+    "/edx/app/edxapp/edx-platform/themes/"
+]
+TEMPLATES[1]["DIRS"] = _make_mako_template_dirs
+derive_settings(__name__)
+
+DEFAULT_SITE_THEME = 'edx.org'
+TAHOE_SILENT_MISSING_CSS_CONFIG = True
```

Comment:
This is meant for running `pytest` not for anything else. So probably it needs a theme and we'll not need this change.

```suggestion
TAHOE_SILENT_MISSING_CSS_CONFIG = True
```